RELEASES.md

```diff
@@ -86,31 +86,39 @@ project layouts.**
 
 ```bash
 python -m pip install \
-  --pre --find-links https://therock-nightly-python.s3.amazonaws.com/gfx94X-dcgpu/index.html \
+  --index-url https://d2awnip2yjpvqn.cloudfront.net/v2/gfx94X-dcgpu/ \
+  rocm[libraries,devel]
+```
+
+#### gfx950-dcgpu
+
+```bash
+python -m pip install \
+  --index-url https://d2awnip2yjpvqn.cloudfront.net/v2/gfx950-dcgpu/ \
   rocm[libraries,devel]
 ```
 
 #### gfx110X-dgpu
 
 ```bash
 python -m pip install \
-  --pre --find-links https://therock-nightly-python.s3.amazonaws.com/gfx110X-dgpu/index.html \
+  --index-url https://d2awnip2yjpvqn.cloudfront.net/v2/gfx110X-dgpu/ \
```

Comment:
Tested this on my dev machine. Much faster download speed from cloudfront (50-70 MB/s) compared to s3 (3 MB/s).

Logs showing install behavior with and without torch, as well as sanity checks: https://gist.github.com/ScottTodd/ef7cc7476793bae2fd1a6245908e1128